openhexa/cli/api.py

```diff
@@ -207,8 +207,17 @@ def upload_pipeline(config, pipeline_directory_path: str):
         click.echo("Generating ZIP file:")
     files = []
     venv_path = None
+    env_vars = get_local_workspace_config(Path(pipeline_directory_path))
+    # default value for files path in workspace.yaml
+    excluded_dirs = ["workspace"]
+    if env_vars.get("WORKSPACE_FILES_PATH"):
+        excluded_dirs.append(Path(env_vars["WORKSPACE_FILES_PATH"]).name)
+
     with ZipFile(zipFile, "w") as zipObj:
         for path in directory.glob("**/*"):
+            # we need to exclude workspace folder and all its content
+            if path.match(f"{excluded_dirs[0]}/*") or path.match(f"{excluded_dirs[1]}/*") or path.name in excluded_dirs:
```

Comment:
@cheikhgwane I think that you can do something more robust here:

- Either `excluded_dirs` is a list and you need to write code that can work with a list of any size
- Or you only want to deal with `workspace` and the path set in `workspace.yaml`, and in that case, just write code that explicitly deals with these two paths